frontend/src/service/levelService.ts

```diff
@@ -1,15 +1,23 @@
+import { LoadLevelResponse } from '@src/models/combined';
+
 import { sendRequest } from './backendService';
+import { getChatMessagesFromDTOResponse } from './chatService';
+import { getDefencesFromDTOs } from './defenceService';
 
-const PATH = 'reset';
+const PATH = 'level/';
```

Comment:
Intentional trailing slash? Seems wrong, even though express lets you get away with it on the backend.